src/components/TodoInput.jsx

```diff
@@ -0,0 +1,41 @@
+import React,{useState} from "react";
+import { v4 as uuidv4 } from "uuid";
+import '../styles/TodoInput.css';
+
+function TodoInput({todos, setTodos }) {
+  const [inputText, setInputText] = useState('');
+    
+  function handleInputChange(event) {
+    setInputText(event.target.value);
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    if (!inputText) {
+      return;
+    }
```

Comment:
The use of handle events are identified and according to the project requirements!